service/src/test/java/uk/gov/hmcts/reform/fpl/utils/TestDataHelper.java

```diff
@@ -46,4 +50,33 @@ public static Placement testPlacement(Element<Child> child) {
             .application(testDocument())
             .build();
     }
+
+    public static Placement testPlacement(Element<Child> child,
+                                          List<PlacementOrderAndNotices> placementOrderAndNotices) {
+        return Placement.builder()
+            .childId(child.getId())
+            .childName(child.getValue().getParty().getFullName())
+            .application(testDocument())
+            .orderAndNotices(placementOrderAndNotices
+                .stream()
+                .map(ElementUtils::element)
+                .collect(toList()))
+            .build();
+    }
+
+    public static PlacementOrderAndNotices testPlacementOrderAndNotices(
+        PlacementOrderAndNotices.PlacementOrderAndNoticesType type, String documentBinaryUrl) {
+        return PlacementOrderAndNotices.builder()
+            .type(type)
+            .document(DocumentReference.builder().binaryUrl(documentBinaryUrl).build())
+            .build();
```

Comment:
could be:

return testPlacementOrderAndNotices(type, DocumentReference.builder().binaryUrl(documentBinaryUrl).build());